src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaCacheClientFetchFactory.java

```diff
@@ -956,6 +963,13 @@ private void doClientReplyDataIfNecessary(
                 {
                     break;
                 }
+
+                if (maximumAge == KafkaAge.HISTORICAL && cursor.offset > initialGroupLatestOffset)
```

Comment:
Let's use static import for `HISTORICAL`.

Does `>` condition work even if there are no live messages after `initialGroupLatestOffset`?

Reply:
Yep that works